Features.md

```diff
@@ -0,0 +1,157 @@
+# This is WIP  - don't rely on it util this notice is gone
+
+Last update: 
+05/22/2019 @ 11:15pm (UTC)
+
+## Features
+What the clients have to implement to be feature complete. This can act as a checklist when implementing a new client.
+*Only those which are already possible with the current core.*
+  
+- Searching
+  - For Chats
+  - For Messages by content
+
+- Chat Managment
+  - Archiving
+  - Deleting
+  - Mute chat notefications
+  - Chat List
+  - Change Name
+  - New Chat
+    - PM / DM chat (chat with one contact)
+    - Group
+    - Verified Group
+  - Groups
+    - Add/remove members
+    - leave a group
+    - Display Group Avatar
+    - Change Group Avatar
+  - Contact Requests
+
+- Out-of-Band Verification
+  - join verified group
+  - verify an contact
+  - Methode
+    - QR-code + message (online)
+
+- Contact Managment
+  - Blocking Contacts
+  - Adding Contacts
+  - Deleting Contacts
+
+- Messages
+  - Text
+    - Emoji Picker
+  - Attachment
+    - Video
+    - Image
+    - Audio
+    - File
+  - Location Attribute
+  - Drafts
+  - Star Message
```

Comment:
> *Only those which are already possible with the current core.*

starring messages are not implemented yet on ios/android and afaik in the core.

Reply:
you're right that is only halfway implemented